include/deal.II/lac/la_parallel_block_vector.h

```diff
@@ -448,6 +448,21 @@ namespace LinearAlgebra
       virtual Number operator* (const VectorSpaceVector<Number> &V) const override;
 
       /**
+       * Calculate scalar product between each block of this vector and @p V
+       * and store the result in a full matrix @p matrix. If @p symmetric is
+       * <code>true</code>, it is assumed that inner product results in a
+       * square symmetric matrix and almost half of the scalar products can be avoided.
```

Comment:
I needed to read the implementation to find out what you are saying here with the matrix. I think it would be useful to make it explicitly that
```
  This function computes the result by forming $A_{ij}=U_i \cdot V_j$ where $U_i$ and $V_i$ indicate the
  $i$th block (not element!) of $U$ and the $j$th block of $V$, respectively.
```